### PR TITLE
Mention increase of JVM memory in IntelliJ How To Page

### PR DESCRIPTION
**Pull request by sgerlach**
Signed-off-by: Scott Gerlach <scott.gerlach@stackhawk.com>

**- Summary**
Gradle needs more memory to run correctly when using IntelliJ IDEA

**- Test plan**

Just a section in the single plage

**- Description for the changelog**

Updated the IntelliJ IDEA getting started with a section to allocate more memory for gradle

RueCollar

### CODE REVIEW

**Review by psiinon — APPROVED** [review]

File: site/content/docs/developer/building-zap-with-intellij-idea.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ This guide explains how to make changes to ZAP using IntelliJ IDEA.
 ## Preparation
 You will need to have followed the [Quick Start Guide to Building ZAP](../quick-start-build/) and installed a version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/).
 
+##### Gradle Resources
+Working with ZAP in IntelliJ IDEA may need a bit more Java resources for the gradle actions. To adjust how much memory gradle can use on your machine set the options in [gradle.properties](https://docs.gradle.org/current/userguide/build_environment.html)
```
Comment (thc202) [inline site/content/docs/developer/building-zap-with-intellij-idea.md:17]: - `gradle` → `Gradle`
 - `options` → `option`

Comment (sgerlach) [reply site/content/docs/developer/building-zap-with-intellij-idea.md]: updated and ready for review

**Review by kingthorin — APPROVED** [review]
Thanks

**Comment by thc202** [comment]
Thank you!

**Pull request merged**